src/styles/style.css

```diff
@@ -43,3 +43,7 @@ input:focus {
 span:focus {
   outline: none;
 }
+
+/* input[type='checkbox']:checked + label.strikethrough {
+  text-decoration: line-through;
+} */
```

Comment:
- [ ] Please, remove this old piece of commented code so the repo gets cleaner 💪